new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -338,6 +340,18 @@ public static boolean installVoyager(VoyagerParams params) {
     return Voyager.install(params);
   }
 
+  /**
+   * Install traefik ingress controller.
```

Comment:
use 'Traefik' since it is a product name

Reply:
fixed